README.asciidoc

```diff
@@ -369,6 +370,29 @@ command names:
 ------------------------------------------------------------------------------------------
 
 
+[[Piping]]
+Hints on using image-bootstrap within a pipe
+--------------------------------------------
+If you want to run *image-bootstrap* in a pipe to capture its output to both
+`stdout/stderr` in a single log file, be sure to run *image-bootstrap* in
```

Comment:
With my picky hat on, would you mind to apply this change twice:
```diff
--- `stdout/stderr`
+++ `stdout`/`stderr` 
```
I believe that's better semantics.

Reply:
ack